### ISSUE TITLE
Add fstrim support

### ISSUE THREAD

**Issue by laurivosandi** [openebs/rawfile-localpv#18]
To reclaim space in the host one should regularly issue `fstrim` on the filesystem. Looks like recent kernels pass this through correctly for loopback deivces as well.

**Comment by tiagolobocastro** [comment]
@laurivosandi are you suggesting rawfile-localpv to send this fstrim command by itself?

**Comment by mhkarimi1383** [comment]
Hi
I think it should be handled using the host OS cronjobs/timers

It can cause system to be slow in some scenarios or under high load

**Comment by tiagolobocastro** [comment]
I think we should use `mountOptions` from the storage class, which we don't today.
Then user is free to enable discard if they want, or configure fstrim service.

**Comment by mhkarimi1383** [comment]
> I think we should use `mountOptions` from the storage class, which we don't today. Then user is free to enable discard if they want, or configure fstrim service.

Yeah, New impl. for mount is ready for that we could just use them
